esp/bindings/SOAP/Platform/soapbind.cpp

```diff
@@ -286,6 +310,8 @@ void CSoapRequestBinding::post(const char *proxy, const char* url, IRpcResponseB
         soapclient.setReadTimeoutSecs(readTimeoutSecs_);
     if (mtls_secret_.length())
         soapclient.setMtlsSecretName(mtls_secret_);
+    if (client_cert_.length() || ca_certs_.length() || accept_self_signed_)
+        soapclient.setSecureSocketConfig(LINK(createSecClientConfig(client_cert_, client_priv_key_, ca_certs_, accept_self_signed_)));
```

Comment:
Leak: LINK shouldn't be required - because the function returns an object with the link count already incremented.

Reply:
Fixed.